lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -6834,11 +6834,19 @@ namespace Js
                     - If the size is >= 100, scale by 1.6
             */
             const uint loopPercentage = GetByteCodeInLoopCount() * 100 / max(1u, GetByteCodeCount());
-            if(loopPercentage <= 50)
+            const int byteCodeSizeThresholdForInlineCandidate = CONFIG_FLAG(LoopInlineThreshold);
```

Comment:
>CONFIG_FLAG(LoopInlineThreshold) [](start = 64, length = 32)

We could probably call a function here that would return the minimum of all the inline thresholds that we have. Just something to keep in mind when you enable this.